packages/components/src/components/es-progression/types.ts

```diff
@@ -6,6 +6,7 @@ export interface Checkpoint {
     id: string;
     title: string;
     disabled?: true;
+    clickable?: boolean;
```

Comment:
I think it's good to have it on a case by case basis, but we should also have a global prop, so you can just set it once.

Also, it feels weird defaulting to true on this..

Lets go for:

`readonly` as a prop:

```
@Prop() readonly?: boolean;
```

Then for we can do:

```
        return this.transformedCheckpoints.flatMap(
            (
                {
                    id,
                    title,
                    disabled = false,
                    readonly = this.readonly ?? false,
                    color,
                    inactiveColor,
                    icon,
                i,
            ) => {
```